Source/Codecov.Tests/Coverage/Tool/CoverageTests.cs

```diff
@@ -10,19 +10,19 @@ namespace Codecov.Tests.Coverage.Tool
 {
     public class CoverageTests
     {
-        [Theory, InlineData(null), InlineData(""), InlineData("./ coverage.xml")]
+        [Theory, InlineData(null), InlineData(""), InlineData("./coverage.xml")]
         public void CoverageReport_Should_Be_Empty_If_The_File_Does_Not_Exits(string fileData)
         {
             // Given
-            var options = Substitute.For<ICoverageOptions>();
+            ICoverageOptions options = Substitute.For<ICoverageOptions>();
```

Comment:
please don't change the existing style used in the library

Reply:
It was new vs2019 code formatter :)

Reply:
Ah, I see.
Is there any configuration file that can be added to the repository that overrides the these style settings? I'm working in Linux, so I don't use Visual Studio 2019 unfortunately.